src/ossie_sccp_codec.erl

```diff
@@ -506,18 +506,18 @@ encode_sccp_msgt(?SCCP_MSGT_XUDT, Params) ->
     Data = proplists:get_value(data, Params),
     DataLen = byte_size(Data),
     OptBin = encode_sccp_opts(Params, [segmentation, importance]),
-    PtrCallingParty = 2+0+1,
-    PtrData = 1+CallingPartyLen+1+1,
-    PtrCalledParty = 3+CallingPartyLen+1+DataLen+1+1,
+    PtrCalledParty = 2+0+1,
+    PtrCallingParty = 1+CalledPartyLen+1+1,
+    PtrData = 3+CalledPartyLen+1+CallingPartyLen+1+1,
```

Comment:
```
PtrCalledParty = 3, % Minimum
PtrCallingParty = PtrCalledParty+CalledPartyLen, % Special case when address indicator = 2#X0000000 then 2 minimum
PtrData = PtrCallingParty+CallingPartyLen,
PtrOpt = 0 or PtrData+DataLen,
```
no?

Reply:
yes, but I didn't really want to change too many things